families/config/sawtooth_config/processor/handler.py

```diff
@@ -318,6 +318,41 @@ def _append_setting_entry(state, key, value):
                     key, old_value, value)
 
 
+def _remove_setting_entry(state, key, value):
```

Comment:
jjason, yes, currently, these are just added to implement the functionality. Still have to add the calls to the functions.